server/data/migrations/20190301162446_users.js

```diff
@@ -8,6 +8,10 @@ exports.up = function (knex, Promise) {
         users
             .string('lastName', 128)
             .notNullable();
+        users
+            .string('userName', 128)
```

Comment:
I asked bruce and I think he said he used username not userName. He said he fix it in his if he did have it captialized